app/src/main/java/com/example/allowancetracker/data/PurchaseDatabase.kt

```diff
@@ -10,7 +10,7 @@ import androidx.sqlite.db.SupportSQLiteDatabase
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.launch
 
-@Database(entities = [Purchase::class], version = 2)
+@Database(entities = [Purchase::class], version = 3)
```

Comment:
😭 thank you for doing this for meeeeee!!!

Reply:
no problemo 😁